src/main/java/PlayerThread.java

```diff
@@ -34,6 +34,11 @@ public final void runPlayer() {
         //Loop prompt
         while(true) {
             Instruction currCommand = player.getInstruction();      //Blocking command waiting for input
+            if(!player.instructionLengthValid()){
```

Comment:
While this does work, it still doesn't abide by the SD paradigms we saw. I **really** suggest throwing an exception and catching it here instead of creating implicit protocols which obscure the functionality of the code.